src/core/js/models/adaptModel.js

```diff
@@ -689,23 +695,23 @@ define([
      * @param {Backbone.Model} model Origin backbone model
      * @param {*} value New property value
      */
-    onAll: function(type, model, value) {
-        if (!_.contains(this.bubblingEvents, type)) return;
+    onAll(type, model, value) {
+        if (!_.result(this, 'bubblingEvents').includes(type)) return;
         var event = new ModelEvent(type, model, value);
         this.bubble(event);
-    },
+    }
 
     /**
      * Internal event handler for bubbling events.
      * @param {ModelEvent} event
      */
-    bubble: function(event) {
+    bubble(event) {
         if (!event.canBubble) return;
         event.addPath(this);
         this.trigger("bubble:" + event.type + " bubble", event);
```

Comment:
Suggest template literal:
```suggestion
        this.trigger(`bubble:${event.type} bubble`, event);
```